src/main/resources/assets/opencomputers/bcbuilder/blocks/adapter.json

```diff
@@ -1,12 +0,0 @@
-[{
```

Comment:
When updating dependencies I couldn't find a Buildcraft version for MC 1.16 so I removed this part as well so as to not leave it lingering. It can probably be re-added but I'd rather know what I'm ensuring compatibility with before I do.

Reply:
BuildCraft is extremely unlikely to come back to Forge. It's acceptable to remove integration here.

Reply:
Ahhh, I was just aware that it was still maintained, but missed the no longer on Forge part. Got it!